app/Http/Controllers/Table/SyslogController.php

```diff
@@ -91,4 +92,40 @@ public function formatItem($syslog)
             'priority' => htmlentities($syslog->priority),
         ];
     }
+
+    private function setLabel($syslog)
+    {
+        $output = "<span class='alert-status ";
+        $output .= $this->priorityLabel($syslog->priority);
+        $output .= "'>";
+        $output .= "</span>";
+
+        return $output;
+    }
+
+    /**
+     * @param int $syslog_priority
+     * @return string $syslog_priority_icon
+     */
+    private function priorityLabel($syslog_priority)
```

Comment:
This should be a common function, if you search for `label-danger` in the code you see how many places does the same thing over and over :/

Reply:
I know, I took it from the EventlogController.

But what is best, one func with lots of conditionals or different functions even doing "the same"?

For example, eventlog handles with numeric "severity" while syslog does by string "priority".

Reply:
@Jellyfrog do you think I have to do a different approach with what I've said about different conditions?